tests/test_modeling_common.py

```diff
@@ -1040,7 +1045,7 @@ def check_hidden_states_output(inputs_dict, config, model_class):
     def test_retain_grad_hidden_states_attentions(self):
         config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
         config.output_hidden_states = True
-        config.output_attentions = True
+        config.output_attentions = True if self.has_attentions else False
```

Comment:
@ydshieh I've taken your comment into account, is this OK?

Reply:
LGTM, thanks!